app/controllers/conversations.py

```diff
@@ -0,0 +1,66 @@
+import json
+import subprocess
+
+from flask import Blueprint, jsonify, current_app
+from sqlalchemy import func
+
+from app import db
+from app.models.conversations import ConversationModel
+from app.schemas.conversations import ConversationSchema, UpdateConversationSchema
+from app.helpers import validate_input
+
+conversations_blueprint = Blueprint("conversations", __name__)
+
+secret_key = current_app.config["SECRET_KEY"]
+
+
+@conversations_blueprint.route("/conversations", methods=["POST"])
+@validate_input(schema=ConversationSchema)
+def create_new_conversation(data):
+    # Create new conversation based on username
+    new_conversation = ConversationModel(username=data['username'])
+    db.session.add(new_conversation)
+    db.session.commit()
+
+    # Run model to have the first question
+    response = execute_models([])
+    return jsonify({"message": response}), 201
```

Comment:
Would be nice if you can have docstring to explain the input and output of the API. I see that you did do it in the `model_helpers.py` file.